src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream.mdx

```diff
@@ -75,9 +75,11 @@ Consider the following when evaluating the cost of the AWS CloudWatch metric str
 * [AWS Kinesis Firehose ingest](https://aws.amazon.com/kinesis/data-firehose/pricing/).
 * [AWS Kinesis Firehose data transfer](https://aws.amazon.com/kinesis/data-firehose/pricing/).
 * Optional: [AWS Config service](https://aws.amazon.com/config/pricing/) used to enrich metrics with resource metadata on select AWS namespaces.
-
+<Callout variant="tip">
+New Relic needs access to AWS Config Service to detect, identify and monitor your AWS services. Disabling the access will lead to an incomplete monitoring and poor New Relic experience.
+</Callout>
```

Comment:
this is an important addition

Reply:
<img width="1151" alt="Screenshot 2023-01-30 at 09 18 54" src="https://user-images.githubusercontent.com/113110223/215424066-513cce42-cad1-46a1-8995-6e88b4c771e7.png">

Reply:
^^ the screenshoot is outdated, please check the source code directly